portfolio/src/main/webapp/index.html

```diff
@@ -24,12 +23,18 @@ <h1 id="title">Lina's Portfolio</h1>
       <p>Click here to get a random fun fact about me:</p>
       <button onclick="addRandomFunFact()">Hello</button>
       <div id="fun-fact-container"></div>
-      <h1>Messages</h1>
-      <p>Click here to see messages from the server:</p>
-      <button onclick="getMessages()">Fetch server messages!</button>
-      <div id="messages-container"></div>
 
-      <form action="/data" method="POST">
+      <h1>Comments:</h1>
+      <label for="comment-limit">Enter the limit on the number of comments you want to display:</label>
+      <input type="number" id="comment-limit" name="comment-limit"
+        min="0" max ="100" value = "10"
```

Comment:
nit: Be consistent with the spacing. Usually in HTML we don't put a space around the =